README.md

```
@@ -1,33 +1,68 @@
     }
-</code>
\ No newline at end of file
+
+    fmt.Printf("Current temperature in London: %f°C\n", currentWeather.Temperature)
+}
+```
+
+## Configuration
+Set up your client with the API key provided by the weather service:
+
+```
```

Markdownlint (0.35.0)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)